crates/subspace-networking/src/node_runner.rs

```diff
@@ -1159,4 +1121,70 @@ where
             }
         }
     }
+
+    async fn get_peers_to_dial(&mut self) -> Vec<PeerAddress> {
+        let mut result_peers = Vec::new();
```

Comment:
Please, pre-allocate all vectors, sets and maps if you know the size. In this case you certainly do.